pages/about/index.vue

```diff
@@ -1,106 +1,73 @@
-            <template #linkPyConBlog2016>
-                <ext-link :href="linkPyConBlog2016" highlight>{{
-                    $t('link.pyConBlog2016')
-                }}</ext-link>
-            </template>
-            <template #linkPhotoAlbum>
-                <ext-link :href="linkPhotoAlbum" highlight>{{
-                    $t('link.photoAlbum')
-                }}</ext-link>
-            </template>
-        </i18n>
-        <i18n path="paragraph.year2017" tag="p"></i18n>
-        <i18n path="paragraph.year2018" tag="p"></i18n>
-        <i18n path="paragraph.year2019" tag="p"></i18n>
-        <i18n path="paragraph.year2020" tag="p"></i18n>
-        <i18n path="paragraph.year2021" tag="p"></i18n>
-        <i18n path="paragraph.outro" tag="p"></i18n>
-    </i18n-page-wrapper>
+        </Banner>
+        <div class="flex flex-wrap justify-center mx-2 my-3">
+            <div v-for="(img, i) in introImgs" :key="`intro_img_${i}`">
+                <div
+                    class="w-32 md:w-56 lg:w-56 h-24 md:h-40 lg:h-40 bg-contain bg-no-repeat mx-2 my-3"
+                    :style="{
+                        'background-image': 'url(' + img + ')',
+                    }"
+                ></div>
+            </div>
+        </div>
+        <i18n-page-wrapper>
+            <p></p>
+        </i18n-page-wrapper>
+    </div>
 </template>
 
 <script>
 import i18n from '@/i18n/about/index.i18n'
 import I18nPageWrapper from '@/components/core/i18n/PageWrapper'
-import ExtLink from '@/components/core/links/ExtLink'
+import Banner from '@/components/core/layout/Banner'
+import CoreH1 from '@/components/core/titles/H1'
+import AboutBanner from '@/static/img/about/Banner.svg'
+import IntroImg1 from '@/static/img/about/Intro-1.png'
+import IntroImg2 from '@/static/img/about/Intro-2.png'
+import IntroImg3 from '@/static/img/about/Intro-3.png'
+import IntroImg4 from '@/static/img/about/Intro-4.png'
 
 export default {
     i18n,
     name: 'PageAbout',
     components: {
         I18nPageWrapper,
-        ExtLink,
+        Banner,
+        CoreH1,
     },
     data() {
         return {
-            linkPyCtw2008: 'http://wiki.python.org.tw/PycTW2008',
-            linkPyCtw2011: 'http://wiki.python.org.tw/PycTW2011',
-            linkEverybodyPays:
-                'http://jessenoller.com/blog/2011/05/25/pycon-everybody-pays',
-            linkYycBlog2012:
-                'http://yyc.solvcon.net/writing/2012/pycontw2012_finale.html',
-            linkYycBlog2013:
-                'http://yyc.solvcon.net/writing/2013/after_pycontw_2013/index.html',
-            linkCheckIo: 'http://www.checkio.org/',
-            linkPyConBlog2016:
-                'http://pycontw.blogspot.tw/2016/04/blog-post.html',
-            linkPhotoAlbum: 'http://www.flickr.com/photos/pycon_tw/albums',
+            aboutBanner: AboutBanner,
+            introImgs: [IntroImg1, IntroImg2, IntroImg3, IntroImg4],
         }
     },
+    computed: {
+        getBanner() {
```

Comment:
Normally we would name the computed instance as a noun rather than treating it as a function prefixing with a verb. Therefore, in this case, consider rename it to `bannerStyle`.